pkg/discovery/endpoint/restapi/model.go

```diff
@@ -17,18 +17,30 @@ type WellKnownResponse struct {
 	OperationEndpoint  string `json:"operationEndpoint,omitempty"`
 }
 
-// WebFingerResponse web finger response.
-type WebFingerResponse struct {
+// JRD is a JSON Resource Descriptor as defined in https://datatracker.ietf.org/doc/html/rfc6415#appendix-A
+// and https://datatracker.ietf.org/doc/html/rfc7033#section-4.4.
+type JRD struct {
 	Subject    string                 `json:"subject,omitempty"`
 	Properties map[string]interface{} `json:"properties,omitempty"`
-	Links      []WebFingerLink        `json:"links,omitempty"`
+	Links      []Link                 `json:"links,omitempty"`
+}
+
+// WebFingerResponse is a JSON Resource Descriptor as defined in
+// https://datatracker.ietf.org/doc/html/rfc6415#appendix-A and
+// https://datatracker.ietf.org/doc/html/rfc7033#section-4.4.
+// TODO: Remove this once the 'replace' issue in the orb-driver is resolved
+// (it creates a dependency on this struct name).
+type WebFingerResponse struct {
```

Comment:
@fqutishat `orb-driver` has a `replace` for Orb, which breaks when I renamed `WebFingerResponse` to `JRD` since `orb-driver` imports `github.com/hyperledger/aries-framework-go-ext/component/vdr/orb` which in turn imports `WebFingerResponse`. Does the replace have to be there? If so, then I'll have to update `aries-framework-go-ext` after this to use the new JRD struct name, then I can go back here and remove the duplicate struct.